components/raftstore/src/store/peer.rs

```diff
@@ -802,7 +875,12 @@ where
     /// 1. Set the region to tombstone;
     /// 2. Clear data;
     /// 3. Notify all pending requests.
-    pub fn destroy<T>(&mut self, ctx: &PollContext<EK, ER, T>, keep_data: bool) -> Result<()> {
+    pub fn destroy(
+        &mut self,
+        engines: &Engines<EK, ER>,
```

Comment:
Any reasons not to use `PollContext`?

Reply:
Because the `PerfContext` needs borrow as mutable and `PollContext` has been borrow as immutable for `StoreMeta`.

Reply:
The `PerfContext` here still belongs to `PollContext `.